app/Http/Middleware/MaintenanceMode.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+namespace Ushahidi\App\Http\Middleware;
+
+use Closure;
+
+class MaintenanceMode
+{
+
+   /**
+    * Handle an incoming request.
+    *
+    * @param  \Illuminate\Http\Request  $request
+    * @param  \Closure  $next
+    * @param  string|null  $guard
+    * @return mixed
+    */
+    public function handle($request, Closure $next, $guard = null)
+    {
+        $maintenanceMode = getenv('MAINTENANCE_MODE');
+        if ($maintenanceMode === '1') {
```

Comment:
It'd probably be better just to check for any truthy value, thats much more normal behaviour for boolean environment vars in laravel